api/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -56,14 +56,20 @@ type NginxIngressControllerSpec struct {
 	// +optional
 	DefaultSSLCertificate *DefaultSSLCertificate `json:"defaultSSLCertificate,omitempty"`
 
+	// DefaultBackendService defines the service that the NginxIngressController should default to when given HTTP traffic with not matching known server names.
+	// The controller directs traffic to the first port of the service. Service should expose /health and / endpoints
```

Comment:
is it /health or /healthz?

is /healthz even necessary? That's likely something more on the Pod deployment that the user should control. The same for `/`, it's on the user to configure how the default Service backing Deployment routes

Reply:
It is /healthz and I'll fix this comment because a lot of it is a bit off

Reply:
actually yeah /healthz and / were directly mention in the docs for this command and used in the example default backend service they linked. I could be reading this wrong though: 
https://kubernetes.github.io/ingress-nginx/user-guide/default-backend/
https://github.com/kubernetes/ingress-nginx/blob/main/images/custom-error-pages/rootfs/main.go

Reply:
how are /healthz and / used by the Ingress Controller? In theory it should just route traffic to the service specified and the service routes to a workload that's configured to understand the health check.

/ will be the route used so that makes sense but why is having a specific health check necessary for this when that really changes at the workload level which doesn't interface with Nginx directly